kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/encrypt/EncryptionDekCache.java

```diff
@@ -39,6 +40,11 @@ public class EncryptionDekCache<K, E> {
 
     public static final int NO_MAX_CACHE_SIZE = -1;
     private CipherSpecResolver cipherSpecResolver;
+    private final AtomicLong invalidationCount = new AtomicLong(0L);
+
+    public long invalidationCount() {
```

Comment:
This is a test only method, right?  We should document it that way, and maybe reduce the visibility to package-protected.